### ISSUE TITLE
HSlider or Gamepad error

### ISSUE THREAD

**Issue by DimitriyPS** [godotengine/godot#59457]
### Godot version

3.5 b2 (no way to check others )

### System information

Linux Mint 20.2, Gamepad 8BitDO SF30 PRO

### Issue description

For standard actions in InputMap: ui_left, ui_right, ui_up, ui_dot - 2 events from the gamepad are assigned to each: D-pad and left stick.

The Slider is used in the interface scene. When it is in focus, its change occurs according to the actions ui_left, ui_right. From the cross it works fine, from the left stick one actuation works after that none command from the gamepad is not accepted by the application... at the same time, the HSlider is still in focus and the same actions on keyboard events work... moreover, the gamepad is no longer processed even if with the mouse  focus is transferred to another control and even if the interface scene is reloaded... 

I can't determine if this is a HSlider or gamepad processing error.

https://user-images.githubusercontent.com/11189223/159821826-9b9a72e7-c027-4687-adb8-df613f08c213.mp4

### Steps to reproduce

As described in " Issue description "

**Comment by Calinou** [comment]
@DimitriyPS Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by DimitriyPS** [comment]
I'll start with the fact that there are no errors and the issue can be closed.

I created a small demo... but it worked without errors. I was in a lot of confusion - in my project there is an error, while in a small demo there is no error. At the same time, Slider is processed by its mechanics by the standard Input Map without my participation.

The reason is somehow related to the fact that I saved the changed settings to disk every time any value was changed. The D-pad is a button - it causes a single operation. The stick is an axis, as it moves from neutral to the stop, it causes multiple triggering ... it seems that multiple triggering accompanied by saving to a file causes some kind of collapse in Godot.

I revised my old approach. It doesn't make sense to reactively save each setting change to a file. Now I've made one call to save the settings to a file when exiting the settings scene to the main menu scene.
